### PR TITLE
Fix package comment

### PR DESCRIPTION
**Pull request by egurnov**
Fixes issue openebs/openebs#2158 by separating license from package comment

**What this PR does / why we need it**: Fixes issue openebs/openebs#2158 by separating license from package comment

**Which issue this PR fixes**: openebs/openebs#2158

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/maya-apiserver/spc-watcher/handler.go [hunk 1]
```diff
@@ -79,20 +80,17 @@ func (c *Controller) spcEventHandler(operation string, spcGot *apis.StoragePoolC
 		}
 
 		return addEvent, err
-		break
```
Comment (egurnov) [inline cmd/maya-apiserver/spc-watcher/handler.go]: go has not fallthrough in switch, break not needed. Especially after a return.

File: cmd/maya-apiserver/spc-watcher/spc_lease_test.go [hunk 2]
```diff
@@ -69,7 +69,7 @@ func PodCreator(fakeKubeClient kubernetes.Interface, podName string) {
 		}
 		_, err := fakeKubeClient.CoreV1().Pods("openebs").Create(podObjet)
 		if err != nil {
-			glog.Errorf("Fake pod object could not be created:", err)
```
Comment (egurnov) [inline cmd/maya-apiserver/spc-watcher/spc_lease_test.go]: No formatting directives, `glog.Error` will do just fine.

File: cmd/maya-apiserver/spc-watcher/storagepool_create.go [hunk 3]
```diff
@@ -57,7 +57,7 @@ func (c *Controller) CreateStoragePool(spcGot *apis.StoragePoolClaim, reSync boo
 		glog.Errorf("Could not acquire lease on spc object:%v", err)
 		return err
 	}
-	glog.Info("Lease acquired successfully on storagepoolclaim %s ", spcGot.Name)
```
Comment (egurnov) [inline cmd/maya-apiserver/spc-watcher/storagepool_create.go]: Formatting directive present, `glog.Infof` should be used instead.

File: cmd/maya-apiserver/spc-watcher/handler.go (hunk 1 above)
Comment (sonasingh46) [reply cmd/maya-apiserver/spc-watcher/handler.go]: Good pick!

File: cmd/maya-apiserver/spc-watcher/handler.go [hunk 4]
```diff
@@ -13,6 +13,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
```
Comment (sonasingh46) [inline cmd/maya-apiserver/spc-watcher/handler.go]: Can you please remove the comment at line 17. Not required.

**Pull request merged**